src/test/ecs/wizards/executeCommand.test.ts

```diff
@@ -0,0 +1,34 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as _ from 'lodash'
+import { createWizardTester, WizardTester } from '../../../test/shared/wizards/wizardTestUtils'
+import { instance, mock } from '../../utilities/mockito'
+import { CommandWizard, CommandWizardState } from '../../../ecs/wizards/executeCommand'
+
+describe('CreateServiceWizard', function () {
+    let tester: WizardTester<CommandWizardState>
+    const node = mock()
```

Comment:
Missing type. We should fix `mock` to use a class prototype to infer the type (separate PR)